test/macros/spawn_lnd_daemon.js

```diff
@@ -0,0 +1,235 @@
+const removeDir = require('rimraf');
+const {spawn} = require('child_process');
+const uuidv4 = require('uuid/v4');
+const asyncAuto = require('async/auto');
+const rpcServerReady = /password RPC server listening on|Finished rescan for 0 addresses/;
```

Comment:
Style is alphabetize and put a newline between requires and other var declarations